spec/factories/users.rb

```diff
@@ -27,6 +27,22 @@
       registration_complete { true }
     end
 
+    trait :closed do
```

Comment:
Please simplify this trait by inheriting from `registered`

Reply:
This trait has now been updated to inherit from `registered`